src/code/z_sub_s.c

```diff
@@ -154,7 +154,9 @@ Gfx* SubS_DrawTransformFlex(GlobalContext* globalCtx, void** skeleton, Vec3s* jo
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013AD9C.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013AED4.s")
+void SubS_UpdateFlags(u16* flags, u16 setBits, u16 unsetBits) {
+    *flags = (*flags & ~unsetBits) | setBits;
+}
```

Comment:
```suggestion
void SubS_UpdateFlags(u16* flags, u16 setBits, u16 unsetBits) {
    *flags &= ~unsetBits;
    *flags |= setBits;
}
```
This wouldn't work would it? Also, do you have a sense as to what these flags do? I'm guessing that will come with the rest of `z_sub_s`?

Reply:
Unfortunately no, that does not work.
To answer your other questions, these flags are not specific to any single system, instead this is a general bitFlags helper function that can be used with any kind of bitFlag. So the flags will depend on what actually use this function.